### PR TITLE
Add 4 system.fs.file_handles.* metrics

### PR DESCRIPTION
**Pull request by remicalixte**
### What does this PR do?

Add :

* `system.fs.file_handles.allocated`
* `system.fs.file_handles.allocated_unused`
* `system.fs.file_handles.used`
* `system.fs.file_handles.max`

### Motivation

These metrics are collected by agent 5

### CODE REVIEW

**Review by derekwbrown — CHANGES_REQUESTED** [review]
Same changes available on Windows?

File: pkg/collector/corechecks/system/file_handles.go [hunk 1]
```diff
@@ -83,7 +83,11 @@ func (c *fhCheck) Run() error {
 	fhInUse := (allocatedFh - allocatedUnusedFh) / maxFh
 	log.Debugf("file handles in use: %f", fhInUse)
```
Comment (olivielpeau) [inline pkg/collector/corechecks/system/file_handles.go]: unrelated to your changes: could you remove this debug log line and the others above? They really look like entries added during development and shouldn't be logged from an actual running Agent IMO

**Review by olivielpeau — DISMISSED** [review]
Just an unrelated nit, otherwise LGTM

**Comment by olivielpeau** [comment]
(if there are similar metrics we could report on Windows, let's create a card for that, this PR is only meant to port metrics that Agent v5 already sends by default on Linux)

File: pkg/collector/corechecks/system/file_handles.go (hunk 1 above)
Comment (remicalixte) [reply pkg/collector/corechecks/system/file_handles.go]: Sure

**Pull request merged**